sort/sort_quicksort.f90

```diff
@@ -0,0 +1,86 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
```

Comment:
This file ends in .f90, but all the other FMS files end in .F90. Is this intentional?

My understanding is that some compilers will not run pre-processor on .f90, but will on .F90, but I may be incorrect.

Reply:
There wasn't any preprocessing, so I assumed it would be find to use `.f90`.   But as mentioned in your other comment, maybe it is not needed.